scripts/sanitise-db.sql

```diff
@@ -337,19 +340,50 @@ $$ LANGUAGE plpgsql;
 
 DO $$
 BEGIN
```

Comment:
Can we make sure that all data blob fields specified in [mappings.py](https://github.com/communitiesuk/funding-service-design-post-award-data-store/blob/main/data_store/controllers/mappings.py#L142) are anonymised, just to take a super-cautious approach? E.g., for `funding` we also need to anonymise `funding_category`, `secured`, `spend_for_reporting_period` and `state`